hbase-shell/src/main/ruby/shell/commands/clear_slowlog_responses.rb

```diff
@@ -0,0 +1,47 @@
+#
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements. See the NOTICE file distributed with this
+# work for additional information regarding copyright ownership. The ASF
+# licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Clear slowlog responses maintained in memory by RegionServers
+
+module Shell
+  module Commands
+    # Clear slowlog responses
+    class ClearSlowlogResponses < Command
+      def help
+        <<-EOF
+Clears SlowLog Responses maintained by each or specific RegionServers.
+Specify array of 'server_name' for specific RS. 'server_name' is
+the host, port plus startcode of a RegionServer.
+e.g.: host187.example.com,60020,1289493121758 (find servername in
+master ui or when you do detailed status in shell)
+
+Examples:
+
+  hbase> clear_slowlog_responses                                     => clears slowlog responses from all RS
+  hbase> clear_slowlog_responses ['server_name1', 'server_name2']    => clears slowlog responses from server_name1,
+                                                                        server_name2
+
+
+        EOF
+      end
+
+      def command(server_names = nil)
+        admin.clear_slowlog_responses(server_names)
+      end
+    end
+  end
+end
```

Comment:
Would an on/off for recording be hard to do with off being default?

Reply:
Should not be an issue to provide on/off feature but should we really provide it? The reason being 255 entries might be significantly less to store in RAM anyways.
But again, having this feature as on/off is also a good idea.

Reply:
Oh but I think if we want to on/off dynamically, then we would need to involve ZNode just like any other `_switch` commands. Should we do that? Or may be do it as part of separate sub-task? What would be your preference?

Reply:
Not sure. on/off would make it easier to commit than an on all the time.

On other hand, dynamic config system not available and wouldn't want you writing one just for this.

Reply:
Sure @saintstack sir, agree with you. Added turn on/off config and kept default as off.